Documentation/UseCases/Index.rst

```diff
@@ -272,3 +272,49 @@ like `https://example.com/downloads/detail/some-download/hz6dFgz9/`, where
 
 .. _custom dimension: https://matomo.org/docs/custom-dimensions/
 .. _setting a custom URL: https://matomo.org/faq/how-to/how-do-i-set-a-custom-url-using-the-matomo-javascript-tracker/
+
+Extending the script tag
+========================
+
+Using tracking tools like Matomo within the European Union need special treatments in order
+to let the customer consent and agree with the tracking. Even Matomo respects the "Do not track" setting from the browser not everybody
+may be aware of it.
+
+Some GDPR tools like klaro.js require special attribute settings within the script tag in order to work.
+
+# The event listener
```

Comment:
Please use

    The event listener
    -----------------------

for the headline for consistency.